### PR TITLE
Pass the 'args' options to the simulator

### PR DESCRIPTION
**Pull request by PanayotCankov**
We need this in NativeScript for iOS to start the app waiting for a debugger.

### CODE REVIEW

**Comment by Fatme** [comment]
:+1:

File: lib/iphone-simulator.ts [hunk 1]
```diff
@@ -146,6 +146,13 @@ export class iPhoneSimulator implements IiPhoneSimulator {
 			}
 		}
 
+		if (options.args) {
+			var args = options.args.trim().split(/\s+/);
```
Comment (teobugslayer) [inline lib/iphone-simulator.ts]: What happens when the user writes "a b" at the command prompt intending to give ios-sim a single parameter containing a space?

Do we need to take care of this case?

Comment (teobugslayer) [inline lib/iphone-simulator.ts]: We discussed this in person, +1 from me

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**